go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	go.etcd.io/bbolt v1.3.5
 	golang.org/x/crypto v0.0.0-20200728195943-123391ffb6de
 	golang.org/x/sync v0.0.0-20201020160332-67f06af15bc9
+	golang.org/x/text v0.3.2
```

Comment:
is this import needed?

Reply:
actually I don't know where is that coming from.... :-/